app/src/main/java/com/eveningoutpost/dexdrip/Services/DexCollectionService.java

```diff
@@ -650,6 +650,10 @@ public synchronized void onServicesDiscovered(BluetoothGatt gatt, int status) {
                 }
                 status("Enabled " + getString(R.string.blukon));
                 static_use_blukon = true; // doesn't ever get unset
+
+
+                Home.setPreferencesInt("bridge_battery", 100);//force battery to max until getting right value
```

Comment:
I think battery should be set to 0 here as 0 implies no value and nothing will be displayed to the user if they open the app while it is set to 0.